### DISCUSSION TITLE
Variants separated by fewer than two amino acids but more than two nucleotides at nucleotide level

### DISCUSSION THREAD

**Discussion by markgene** [HGVSnomenclature/hgvs-nomenclature#144] [Q&A] [answered]
Hi there,

I see the example below in the [SVD-WG010](https://hgvs-nomenclature.org/stable/consultation/SVD-WG010/) example section:

> NP_003997.1:p.Ser988_Gln990delinsAlaLeuHis: variants p.Ser988 and p.Gln990 (LRG_199t1:c.[2962T>G;2970A>T]) are separated by fewer than two amino acids and described as a "delins" variant, not as p.[Ser988Ala;Gln990His]

Just want to double check if we should describe the variant(s) at DNA, RNA and protein level as such: `LRG_199:g.[875912T>G;875920A>T]` `LRG_199t1:c.[2962T>G;2970A>T]` `LRG_199(t1):r.([2962u>g;2970a>u])` `NP_003997.1:p.Ser988_Gln990delinsAlaLeuHis`? So, in this way, we describe the variant individually at nucleotide level and as one single delins at protein level. Right?

**Comment by jfjlaros** [comment]
This seems correct to me, it also in line with what Mutalyzer currently [suggests](https://mutalyzer.nl/normalizer/LRG_199(t1):c.%5B2962T%3EG;2970A%3ET%5D).

Please note though that there is a [discussion](https://hgvs-nomenclature.org/stable/consultation/SVD-WG010/) about when variants should be merged or split, so these recommendations may change in the future.

**Comment by jtdendunnen** [comment, marked as answer]
Indeed, the descriptions you give are correct. The reason between this is that we try to give recommendations that are the same at DNA, RNA and protein level: "variants separated by fewer than two residues (nucleotides/amino acids) are described as a "delins" variant".
